pytest.ini

```diff
@@ -2,6 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 [pytest]
+norecursedirs =
+    cuda_bindings/examples
+    cuda_core/examples
```

Comment:
Q: What does this do?

Reply:
`*_test.py` is a matched-by-default pattern as well https://docs.pytest.org/en/stable/explanation/goodpractices.html#conventions-for-python-test-discovery